src/testHelper.js

```diff
@@ -1,6 +1,7 @@
 import dotenv from 'dotenv';
 import {createTestUser, removeTestUser} from '@webex/test-users';
 import Webex from 'webex';
+import {constructHydraId} from '@webex/common/dist/uuid-utils';
```

Comment:
Are you importing like this because of the export issue on the SDK for `@webex/common`?

Reply:
No what do you mean? That’s how we export them regularly. And also we not bundling this code.

Reply:
To me a "regular" import looks like:
```suggestion
import {constructHydraId} from '@webex/common';
```
We don't directly import from files

Reply:
Oh shit. I didn’t even see that